specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-01-31-preview/types/applications.json

```diff
@@ -0,0 +1,360 @@
+    "version": "2023-01-31-preview"
+  },
+  "paths": {},
+  "definitions": {
+    "application": {
+      "type": "object",
+      "description": "Application tracked resource.",
+      "allOf": [
+        {
+          "$ref": "../../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/applicationProperties",
+          "x-ms-client-flatten": true,
+          "description": "The resource-specific properties for this resource."
+        }
+      },
+      "required": [
+        "properties"
+      ]
+    },
+    "applicationUpdate": {
+      "description": "An update to an application.",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "applicationProperties": {
+      "type": "object",
+      "properties": {
+        "identities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/UserAssignedManagedIdentityProperties"
+          },
+          "x-ms-identifiers": [],
+          "x-cadl-name": "UserAssignedManagedIdentityProperties[]",
+          "description": "list of User assigned managed identities used by application"
+        },
+        "deployedPlatform": {
+          "$ref": "#/definitions/platformProperties",
+          "description": "Platform on which application will be deployed"
+        },
+        "profileId": {
+          "type": "string",
+          "description": "Chaos Studio profile id of an application resource."
+        },
+        "faultMetadata": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/metadata"
+          },
+          "description": "List of fault metadata of an application resource"
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The status of the last operation.",
+          "readOnly": true
+        }
+      },
+      "description": "The properties of an application resource",
+      "required": [
+        "identities"
+      ]
+    },
+    "applicationListResult": {
+      "type": "object",
+      "description": "Model that represents a list of Application resources and a link for pagination.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/application"
+          },
+          "description": "List of Application resources.",
+          "readOnly": true
+        },
+        "nextLink": {
+          "$ref": "./common.json#/definitions/urlNullable",
+          "description": "URL to retrieve the next page of Application resources.",
+          "readOnly": true
+        }
+      },
+      "additionalProperties": false
```

Comment:
>    "additionalProperties": false [](http://example.com/codeflow?start=2&length=33)

nit, additional properties false is the default, so you can leave it out, I think

Reply:
removed additionalProperties: false